contentcuration/contentcuration/viewsets/websockets/signals.py

```diff
@@ -0,0 +1,80 @@
+import logging as logger
+
+from asgiref.sync import async_to_sync
+from channels.layers import get_channel_layer
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from contentcuration.models import Change
+from contentcuration.utils.sentry import report_exception
+from contentcuration.utils.websocket_helper import NoneCreatedByIdError
+
+logging = logger.getLogger(__name__)
+
+
+@receiver(post_save, sender=Change, weak=False)
+def broadcast_new_change_model_handler(sender, instance, created, **kwargs):
+    broadcast_new_change_model(instance)
+
+
+def broadcast_new_change_model(instance):
+    channel_layer = get_channel_layer()
+    serialized_change_object = Change.serialize(instance)
+    # Name of channel group
+    room_group_name = instance.channel_id
+
+    # name of indiviual_user group
+    indiviual_room_group_name = instance.user_id
+
+    if(instance.created_by_id is None):
```

Comment:
Why the parentheses?

Reply:
sorry sometimes the java habit kicks in :sweat_smile: